src/functions/readReplayValue.js

```diff
@@ -0,0 +1,26 @@
+import { Console } from '@woowacourse/mission-utils';
+import { ONE_OR_TWO_REGEX } from '../constant/regex.js';
+import { ERROR_MESSAGE } from '../constant/message.js';
+
+/**
+ * @param {string} message
+ * @returns {boolean}
+ */
+const readReplayValue = async function readReplayValueFromUser(message) {
+  const willReplay = await Console.readLineAsync(message);
+  const isValid = ONE_OR_TWO_REGEX.test(willReplay);
+
+  if (!isValid) {
+    throw Error(ERROR_MESSAGE.invalidInput);
+  }
+
+  if (willReplay === '1') {
+    return false;
+  } else if (willReplay === '2') {
+    return true;
+  }
```

Comment:
'1'과 '2' 같은 상수는 묶어서 상수화를 하는 것이 좋습니다!
하나의 파일에서 관리할 수 있고, 이름을 명시해서 해당 상수가 어떤 역할을 하는지 직관적으로 이해하기도 쉬워요.

```js
const GAME = {
  REPLAY: '1',
  QUIT: '2',
};
```

```suggestion
  if (willReplay === GAME.REPALY) {
    return false;
  } else if (willReplay === GAME.QUIT) {
    return true;
  }
```

Reply:
감사합니다!